### PR TITLE
Support for multiple branches

### PR DESCRIPTION
**Pull request by omichel**
This PR aims at extending the action to support the generation of animations for multiple branches and not just the master branch.

### CODE REVIEW

**Comment by omichel** [comment]
The https://github.com/cyberbotics/webots-animation-template repo should be modified accordingly once this PR is merged.

File: entrypoint.sh [hunk 1]
```diff
@@ -10,13 +10,15 @@ python3 /bin/scripts/generate_animation.py
 # Publish animation to `gh-pages` branch
 NAME=$(curl https://api.github.com/users/${GITHUB_ACTOR} | jq -r .name)
 ID=$(curl https://api.github.com/users/${GITHUB_ACTOR} | jq -r .id)
+BRANCH=${GITHUB_REF##*/}
 git config --global user.name "${NAME}"
 git config --global user.email "${ID}+${GITHUB_ACTOR}@users.noreply.github.com"
 git reset --hard
 git fetch
 git checkout gh-pages || git checkout -b gh-pages
 rm -rf $(ls -aI '.git') 2> /dev/null
```
Comment (lukicdarkoo) [inline entrypoint.sh]: I didn't test this, but to me, it seems this line will delete the content of other branches in `gh-pages`. It should be something like:
```bash
[ -d ${BRACNH} ] && rm -rf ${BRANCH}/*
```

Comment (omichel) [reply entrypoint.sh]: Why not simply:
```bash
rm -rf ${BRANCH}
```

**Comment by lukicdarkoo** [comment]
We should generate `index.html` in the root with links to all branches because GitHub Pages doesn't allow dictionary listing. This, we can do before a proper AJAX implementation for branch switching.

File: entrypoint.sh (hunk 1 above)
Comment (lukicdarkoo) [reply entrypoint.sh]: That is much better :D

**Comment by omichel** [comment]
OK, I just created simple index.html file for now.

**Comment by omichel** [comment]
OK, that seems to work: https://cyberbotics.github.io/webots-animation-template/

**Review by lukicdarkoo — APPROVED** [review]
Perfect, thanks!

**Pull request merged**